pkgs/development/python-modules/pypager/default.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  prompt-toolkit,
+  pygments,
+}:
+
+buildPythonPackage {
+  pname = "pypager";
+  version = "3.0.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "prompt-toolkit";
+    repo = "pypager";
+    rev = "0255d59a14ffba81c3842ef570c96c8dfee91e8e";
+    hash = "sha256-uPpVAI12INKFZDiTQdzQ0dhWCBAGeu0488zZDEV22mU=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    prompt-toolkit
+    pygments
+  ];
+
+  pythonImportsCheck = [ "pypager" ];
+
```

```suggestion
  # no tests
  doCheck = false;
```
We highly recommend that you enable checkPhase in the python package.
So, if you don't want to run tests, please explicitly disable it with a reason.

> It is highly encouraged to have testing as part of the package build. This helps to avoid situations where the package was able to build and install, but is not usable at runtime.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#testing-python-packages-testing-python-packages